exampleSite/content/basics/customization/_index.en.md

```diff
@@ -14,8 +14,8 @@ This theme defines the following partials :
 - `footer.html`: the footer of the page. See [output-formats](#output-formats)
 - `body.html`: the body of the page. The body may contain of one or many articles. See [output-formats](#output-formats)
 - `article.html`: the output for a single article, can contain elements around your content. See [output-formats](#output-formats)
-- `menu.html`: left menu. _Not meant to be overwritten_
-- `search.html`: search box. _Not meant to be overwritten_
+- `menu.html`: left menu. *Not meant to be overwritten*
```

Comment:
Personally I prefer underscores for italics while I am using double stars for bold. In my opinion this makes it easier to read, especially once you want to apply both styles.

Nevertheless I inherited the docs in the initial fork which explains why the docs do not adhere to my own standards yet.

Usually when changing this, I am going full circle and change the whole project to this style.

Reply:
Testing the waters here to get a feel for how you handle contributions and what you'll accept. Since I was already in this file for #553, I saw an unrelated small opportunity for improvement and acted on it. Reporting things like linter warnings feels nit picky and a suboptimal use of time to me. Anyway, I didn't want to invest more if you're not into contributions like these. 

Also, fair enough on the preference. I prefer the inverse (obviously), primarily because I've run into a few projects over the years that don't support the underscore variant, such as terraform-docs.